src/lib/index.tsx

```diff
@@ -10,8 +10,8 @@ import UserCard from './containers/UserCard'
 import Login from './containers/Login'
 import CardContainer from './containers/CardContainer'
 import CardContainerLogic from './containers/CardContainerLogic'
-import './style/Cards.scss'
-import './style/Portal.scss'
+import './style/Cards.css'
```

Comment:
node-sass up above doesn't actually fix the styling, its just recommended in the react docs. And even though .css is imported instead of .scss, scss still gets used during development since the file gets watched and written to css